package.json

```diff
@@ -294,6 +294,12 @@
                             ],
                             "description": "Set the smart code completion for CRDs. This would always prevent/allow the plugin to fetch all custom schemas from all CRDs on cluster, despite of their number."
                         },
+                        "vs-kubernetes.supress-output": {
+                            "type": "string",
```

Comment:
💡Question please: just something minor these 2 `types` is intention or is this meant to be just `boolean`. I think `Duplicate object key will not be allowed."